news/2017-11-05-Official-Mapping-Contests:-osu!-Mapping-Olympiad.md

```diff
@@ -0,0 +1,71 @@
+### Songs (STILL NEED HELP WITH THE SONG PREVIEWS)
+
+To start off, we are launching this contest with not one, but **three** songs for you to tackle! Showcasing a variety of our Featured Artists, have yourself a listen to the tracks you’ll be taking on:
+
+- Kuba Oms - Beautiful Uncertainty (Redux)
+- OISHII - Honey Snowflakes
+- cySmix - Escapism
+
+There’s going to be something for everyone’s taste - funky rhythms by Kuba Oms, fast-paced goodness by OISHII and some chill beats by cYsmix!
+
+### Ruleset
+
+- This contest is for the **osu!** (standard) mode only.
+- Each entry should follow the Ranking Criteria.
+- Beatmap submissions may contain files up to 15 MB.
+- Storyboarding will not be considered in the judging.
+- You may submit **one** difficulty for **one** song. Multiple submissions for multiple songs will not be accepted.
+- You may collaborate if you like, but only the submitter of each entry will receive credit for judging and prizes.
+- Maps submitted to the contest must not be uploaded to the BSS until the contest has concluded and the winners have been announced.
+- **Send your finished beatmap as .osu-file to XXX before the deadline expired. Any submissions received after the deadline expired will not be accepted.**
+
+### Judging Criteria
+
+We have also slightly reworked the judging criteria to provide a more reasonable judging base. Down below you can see what your map will be judged for:
+
+- Musical Relevance (10P) - How well does the entry follow the music with its patterns, its structure and overall concept?
+- Creativity (10P) - How unique and/or innovative is the entry in its use of concepts and overall design?
+- Technique (10P) - How well does the entry utilize the editor and its tools in its creation? 
+- Impression (10P) - How does the judge view the map in its entirety?
+- Hitsounding (5P) - How well does the entry utilize hitsounding to emphasize and follow the music?
+
+**Speaking of Judging: We are still looking for capable judges! Are you interested? Feel free to drop a PM to Irreversible with your motivation, knowledge and possible references, where you've already judged. As a compensation, you will receive 2 months of supporter. Please be aware that we can only accept people who do have enough ressources (knowledge, time, reliability) to successfully do this.**
+
+### Prizes
+
+After the entries have been judged and the results finalized, each of the top three maps for each track will receive the following prizes:
+
+- **1st place**: 6 months of osu!supporter, <Olympiad> Winner Badge
+- **2nd place**: 4 months of osu!supporter, <Olympiad> Honorable Mention Badge
+- **3rd place**: 2 months of osu!supporter, <Olympiad> Honorable Mention Badge
+
+Now, you may be wondering why there is a lack of *Elite Mapper* forum title listed in the prizes. After much deliberation, we felt that granting the *Elite Mapper* title to each song’s top placing entrant would not be doing the prestige of the title enough justice. 
```

Comment:
Mentioning this is fine, but it's not a very positive thing to say, without you explicitly stating it no one could be expecting elite mapper as a reward anyways because you didn't list it so i think this just dampens the mood and should either be said positively or not said at all